src/iuse.cpp

```diff
@@ -6816,10 +6816,8 @@ int iuse::remoteveh(player *p, item *it, bool t, const tripoint &pos)
         if( !veh->engine_on ) {
             veh->start_engines();
         }
-    } else if( choice == 3 ) {
-        veh->use_controls( pos );
     } else {
```

Comment:
Removing the return statement that will never happen is good, but I'd prefer to keep ```} else if( choice == 2 ) {``` because it's more obvious what's happening.  The compiler should be able to optomise this as well.